avocado/utils/memory.py

```diff
@@ -48,6 +48,13 @@ def memtotal():
     return read_from_meminfo('MemTotal')
 
 
+def availablememtotal():
+    """
+    Read ``MemAvailable`` from meminfo.
+    """
+    return read_from_meminfo('MemAvailable')
```

Comment:
Ok @ldoktor. I proposed this only for the sake of consistency with `MemTotal` and `freeMemTotal` methods. But generally agree with you - those methods are not necessary. I can close this PR and accordingly modify https://github.com/avocado-framework/avocado-vt/pull/948.